posthog/warehouse/models/external_table_definitions.py

```diff
@@ -613,3 +613,22 @@
         "only_private_comments": BooleanDatabaseField(name="only_private_comments"),
     },
 }
+
+HOGQL_FIELD_DLT_TYPE_MAP = {
+    StringDatabaseField: "text",
+    IntegerDatabaseField: "bigint",
+    BooleanDatabaseField: "bool",
+    DateTimeDatabaseField: "timestamp",
+    StringJSONDatabaseField: "complex",
+}
+
+
+def get_dlt_mapping_for_external_table(table):
+    return {
+        field_name.removeprefix("__"): {
```

Comment:
You can use `field.name` here instead and not worry about the prefix - there's no guarantee that fields will start with `__`, but there is a guarantee that `field.name` will be the DB column name